### ISSUE TITLE
Flood protection on getstatus?

### ISSUE THREAD

**Issue by bbashy** [callofduty4x/CoD4x_Server#109]
I'm using sockets to receive the data from the server (sv_hostname and player count) but it's returning "Resource temporarily unavailable". With 1.7 it was fine and I could send 3/4 requests at the same time but 1.8 I can't.

Are there any limits on the `\xFF\xFF\xFF\xFFgetstatus\x00` response?

**Comment by bbashy** [comment]
Any info on this?

**Comment by IceNinjaman** [comment]
You have called me a "cunt" or whatever insult it was on your nice forum at http://myns.co
Why would you expect an answer from us now?

**Comment by bbashy** [comment]
Not sure what I said but this all started when it was impossible to keep my servers online with people getting mistakenly banned from you releasing a tool to use any GUID - including my own.

You know what you did as well with spamming our servers.

As long as I know you don't want to help, I'll go elsewhere.

**Comment by Kingwolf123** [comment]
You have enraged the mighty !
Now face their wrath !

**Comment by IceNinjaman** [comment]
I have spammed your server? Not sure what you mean. The only thing I personally did was downloading your special Promod scripts. But that you didn't know anyway I assume.
True I released that tool you are talking about. Still no need for hate.

About your question: There is a limit for queries in a time. getstatus queries have a higher penalty than getinfo queries. I think you know unlimited getstatus/getinfo queries were abused to DDoS other services?
